tests/integration/testapp/app.py

```diff
@@ -73,6 +73,17 @@ def form_encoded():
     }
 
 
+@app.route('/binary', methods=['POST'],
```

Comment:
Can you add an integration test that exercises the `.api.binary_types` functionality?